molecule/default/playbook.yml

```diff
@@ -7,7 +7,7 @@
 
     - role: ansible-role-omero-web-apps
       omero_web_apps_packages:
-        - omero-mapr
+        - "omero-mapr==0.3.0a1"
```

Comment:
Let's see if we can get a proper release out in a reasonable time....

Reply:
I assume we'll want a non pre-release for `prod67` anyways